### ISSUE TITLE
Twitter SignIn - SessionStorage Error

### ISSUE THREAD

**Issue by Dave181295** [OAuthSwift/OAuthSwift#708]
### OAuth Version:
- [ x] Version 1
- [ ] Version 2

### OS (Please fill the version) :
- [x] iOS :
- [ ] OSX :
- [ ] TVOS :
- [ ] WatchOS  :

### Installation method:
- [ ] Carthage
- [ ] CocoaPods
- [ x] Swift Package Manager
- [ ] Manually

### Library version:
- [ ] head
- [ ] v2.1.0
- [ ] v2.0.0
- [ ] v1.4.1
- [x (2.2.0) ] other: (Please fill in the version you are using.)

### Xcode version:
- [ ] 11.4 (Swift 5.2)
- [ ] 11.x (Swift 5.1)
- [ ] 10.x (Swift 5.0)
- [14 ] other: (Please fill in the version you are using.)

- [ ] objective c

Whenever trying to login , the safari opens inApp and then we got the error:

```
Unable to process request due to missing initial state. This may happen if browser sessionStorage is inaccessible or accidentally cleared.
```

**Code**

```

 oauthSwift = OAuth1Swift(
            consumerKey:    consumerKey,
            consumerSecret: consumerSecret,
            requestTokenUrl: "https://api.twitter.com/oauth/request_token",
            authorizeUrl:    "https://api.twitter.com/oauth/authorize",
            accessTokenUrl:  "https://api.twitter.com/oauth/access_token"
            
        )
       
        oauthSwift.authorizeURLHandler = SafariURLHandler(viewController: parent, oauthSwift: oauthSwift)
        
```

**Comment by bharatlal087** [comment]
Is there any progress on this issue?

**Comment by phimage** [comment]
nope, code provided is not enought, not authorize call etc... this could be your callback url that not matching one defined in twitter app declaration
then maybe twitter go to oauth 2, because missing state is somethings that I see more with this protocol version

**Comment by Dave181295** [comment]
> nope, code provided is not enought, not authorize call etc... this could be your callback url that not matching one defined in twitter app declaration
> then maybe twitter go to oauth 2, because missing state is somethings that I see more with this protocol version

```
private let callbackURL = "https://appname.firebaseapp.com/__/auth/handler"
```

the same callbackUrl is applied in twitter dashboard, tried with our custom domain and still getting the same error.
